beacon-chain/blockchain/process_block.go

```diff
@@ -255,13 +251,13 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.ReadOnlySi
 			JustifiedCheckpoint: jCheckpoints[i],
 			FinalizedCheckpoint: fCheckpoints[i]}
 		pendingNodes[len(blks)-i-1] = args
-		if err := s.saveInitSyncBlock(ctx, blockRoots[i], b); err != nil {
+		if err := s.saveInitSyncBlock(ctx, root, b); err != nil {
```

Comment:
This function can now be modified to not take the root and use b itself.

Reply:
I guess more on changing the block type from `SignedBeaconBlock` to `RoBlock` across initial sync